src/configuration/app/storage.md

```diff
@@ -60,3 +60,58 @@ web:
 ```
 
 See the [web locations](/configuration/app/web.md) documentation for more details.
+
+## Using the Platform.sh CLI with mounts
```

Comment:
Let's move this section up above the "How do I?" sections.  Those should come last on each page.

Reply:
Then I'd have to change the examples too: currently I have `For example, to download files from the 'private' mount`...